api/pie/v1alpha1/pieprobe_types.go

```diff
@@ -17,9 +17,13 @@ type PieProbeSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="monitoringStorageClass is immutable"
 	MonitoringStorageClass string `json:"monitoringStorageClass"`
 
-	NodeSelector   corev1.NodeSelector `json:"nodeSelector"`
-	ProbePeriod    int                 `json:"probePeriod"`
-	ProbeThreshold string              `json:"probeThreshold"`
+	NodeSelector corev1.NodeSelector `json:"nodeSelector"`
+
+	//+kubebuilder:validation:Maximum:=59
```

Comment:
It would be preferable to set default values for `ProbeThreshold` and `ProbePeriod`, similar to how their command-line argument counterparts had default values. I fixed my code: 8aed78935ccef48da1359053dd11511d2a17d0c9 36f60e1a6c0ab51af76c95bde1853fc509e5a72c